PubHubs/Issues/account/description.xml

```diff
@@ -0,0 +1,55 @@
+<IssueSpecification version="4">
+	<Name>
+		<en>Demo Account</en>
+		<nl>Demo Account</nl>
+	</Name>
+	<ShortName>
+		<en>Demo Account</en>
+		<nl>Demo Account</nl>
+	</ShortName>
+	<SchemeManager>irma-demo</SchemeManager>
+	<IssuerID>PubHubs</IssuerID>
+	<CredentialID>account</CredentialID>
+	<Description>
+		<en>Your PubHubs account</en>
+		<nl>Je PubHubs account</nl>
+	</Description>
+	<ShouldBeSingleton>false</ShouldBeSingleton>
+	<IssueURL>
+		<en>https://privacybydesign.foundation/attribute-index/en/irma-demo.PubHubs.account.html</en>
+		<nl>https://privacybydesign.foundation/attribute-index/nl/irma-demo.PubHubs.account.html</nl>
```

Comment:
is het geen probleem dat deze urls nog een 404 geven?

Reply:
Dat vind ik voor nu niet zo heel erg